lib/caps/ADL/iHD

```diff
@@ -34,7 +34,7 @@ caps = dict(
     hevc_8  = dict(
       maxres    = res8k,
       fmts      = ["NV12", "AYUV"],
-      features  = dict(scc = True),
+      features  = dict(scc = True, mainstill = True),
```

Comment:
Let's abbreviate `mainstill` as `msp` instead.  You will need to update your previous patches.

Reply:
Ok, updated